src/lib/y2storage/proposal/autoinst_devices_creator.rb

```diff
@@ -323,6 +350,45 @@ def set_up_lvm(vgs, devs_to_reuse)
         end
       end
 
+      # Creates Btrfs filesystems in the given devicegraph
```

Comment:
BTW, _in the given devicegraph_ is a bit confusing here. The method do not receive any devicegraph as param. At least, not in a direct way.